src/python_pachyderm/mixin/auth.py

```diff
@@ -18,14 +18,24 @@ def activate_auth(self, subject, github_token=None):
           Pachyderm will generate a new token for the robot user; this token
           will be the only way to administer this cluster until more admins
           are added.
-        * An optional string. This is the token returned by GitHub and used to
-        authenticate the caller. When Pachyderm is deployed locally, setting
-        this value to a given string will automatically authenticate the
-        caller as a GitHub user whose username is that string (unless this
-        "looks like" a GitHub access code, in which case Pachyderm does
-        retrieve the corresponding GitHub username)
-        """
-        return self._req(Service.AUTH, "Activate", subject=subject, github_token=github_token).pach_token
+        * `github_token`: An optional string. This is the token returned by
+          GitHub and used to authenticate the caller. When Pachyderm is deployed
+          locally, setting this value to a given string will automatically
+          authenticate the caller as a GitHub user whose username is that string
+          (unless this "looks like" a GitHub access code, in which case
+          Pachyderm does retrieve the corresponding GitHub username)
+        * `root_token`: An optional string. If specified, this string becomes
+          the Pachyderm cluster root user's token. Currently this is only used
+          for testing and Pachyderm internals (migration), so we're avoiding
+          support for this field in the python-pachyderm client until we find a
+          use for them (feel free to file an issue in
+          github.com/pachyderm/pachyderm)
+        """
+        # TODO(msteffen): Add support for branch triggers
+        if trigger is not None:
+            raise NotImplementedError('branch triggers are not supported yet.')
```

Comment:
This seems like a merge artifact?

Reply:
This was actually a copy and paste error. The intent (which I actually meant to ask for your feedback on) was not to support the `root_token` argument in python_pachyderm. My intuition was that since that was mostly for extract/restore and testing (similar to `extract/restore_auth_token()`), it would be safer to leave it out.

I fixed this error block, though if you think it makes the most sense to just remove it and support `root_token` in python-pachyderm, I'm happy to do that as well.

Reply:
(on doing another pass, I realized there were a handful of careless errors in this PR—not passing arguments through correctly and things like that. I proofread it, though, and _think_ it should be in good shape now)